JitsiConference.js

```diff
@@ -786,6 +791,11 @@ function (jingleSession, jingleOffer, now) {
     // Accept incoming call
     this.room.setJingleSession(jingleSession);
     this.room.connectionTimes["session.initiate"] = now;
+    // Log "session.restart"
+    if (this.wasStopped) {
+        Statistics.analytics.sendEvent("session.restart", now);
```

Comment:
Can't you use `Statistics.sendEventToAll` here instead of calling analytics and callstats separately?